FWCore/Framework/test/stubs/TestTriggerNames.cc

```diff
@@ -224,12 +224,13 @@ namespace edmtest {
       }
     }
 
-    edm::TriggerResultsByName resultsByNameHLT = e.triggerResultsByName("HLT");
-    if(resultsByNameHLT.isValid() && expectedTriggerResultsHLT_.size() > 0) {
+    edm::InputTag tag("TriggerResults", "", "HLT");
+    edm::Handle<edm::TriggerResults> hTriggerResults;
+    e.getByLabel(tag, hTriggerResults);
```

Comment:
This a module used only for tests.  It has both a call to getManyByType and also getByLabel, both getting the TriggerResults type. There is a consumesMany call that allows both of those to work. I am not sure what happens if you call both consumes to get a token and also consumesMany for the same type.